src/components/slider/Slider_card.jsx

```diff
@@ -6,7 +6,7 @@ export default function Slider_card({ name, photo_link, rent, id }) {
   return(
     <div className="card_wrapper mx-10">
       <div className="card_img">
-        <Link to={`/parachutes/${id}`}>
+        <Link to={`/parachutes/${id-1}`}>
           <img src={photo_link} alt="Parachute image" />
```

Comment:
In line # 9, id = id - 1, it means we could change the id by -1. it is not the real id from database. please kindly keep the url based on each card id.